src/atomate2/vasp/jobs/ferroelectric.py

```diff
@@ -139,7 +139,7 @@ def add_interpolation_flow(
     outputs = {}
 
     for i, interp_structure in enumerate(interp_structures[1:-1]):
-        lcalcpol_maker.write_additional_data[""] = {
+        lcalcpol_maker.write_additional_data["structures:json"] = {
             "st_polar": interp_structures[0],
             "st_nonpolar": interp_structures[-1],
             "st_interp_idx": i + 1,
```

Comment:
Question: may I add the interpolated structure directly instead of its index?

Reply:
I think it makes sense to only keep the index to keep track. The structure will be available anyway as the input.